### DISCUSSION TITLE
Registry Location

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8549] [Q&A] [answered]
Hello All,

After installation of KeePassXC where in the registry we can see the values ?

I mean what would be the path in registry ?

Eg: Whenever we are installing any application it is generating the certain entries in registry, Can someone please help me for the KeePassXC.

For Eg: Under HKLM---Software---Classes and there is an entry available, Likewise I wanted to know all the location in registry.

Thanks,

**Comment by droidmonkey** [comment, marked as answer]
Windows stores uninstall information here: https://learn.microsoft.com/en-us/windows/win32/msi/uninstall-registry-key

We also write some values to the registry depending on what you use in the program. The auto start setting is written to the startup programs key, browser integration writes to the specific browser key to point to the native messaging location, and we have a couple values under HKLM/Software/KeePassXC
